From 96591d5bcae197ead4ee8822a16acbeebdd6005e Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:37:42 -0400
Subject: [alsa] trident: Remove unused variable in trident_memory.c

Message-id: <140972823573-371-patchset-jkysela@redhat.com>
Patchwork-id: 91067
O-Subject: [RHEL 7.1 PATCH 370/442] ALSA: trident: Remove unused variable in trident_memory.c
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

'prev' is not used in the function. Remove it.

Signed-off-by: Sachin Kamat <sachin.kamat@samsung.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: e2ff8406ade81f5f431a3cac6ef96720c37906cd)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/trident/trident_memory.c b/sound/pci/trident/trident_memory.c
index 3102a57..04c4746 100644
--- a/sound/pci/trident/trident_memory.c
+++ b/sound/pci/trident/trident_memory.c
@@ -139,12 +139,11 @@ static inline void *offset_ptr(struct snd_trident *trident, int offset)
 static struct snd_util_memblk *
 search_empty(struct snd_util_memhdr *hdr, int size)
 {
- struct snd_util_memblk *blk, *prev;
+ struct snd_util_memblk *blk;
  int page, psize;
  struct list_head *p;
 
  psize = get_aligned_page(size + ALIGN_PAGE_SIZE -1);
- prev = NULL;
  page = 0;
  list_for_each(p, &hdr->block) {
   blk = list_entry(p, struct snd_util_memblk, list);
-- 
1.7.1